SETUP/UNICODE.md

```diff
@@ -54,6 +54,7 @@ The code currently ships with the following character suites:
 * Semitic and Indic transcriptions - Romanized forms of Arabic, Hebrew, Sanskrit
 * Symbols collection - Astronomical, Music, Zodiac and Apothecary symbols
 * Math symbols - Commonly used mathematical symbols not in Basic Latin
+* Basic Cyrillic
```

Comment:
Is it worth saying
```
 * Basic Cyrillic - Russian including pre-revolutionary characters
```
to explain which Cyrillic characters are included

Reply:
> Is it worth saying
> 
> ```
>  * Basic Cyrillic - Russian including pre-revolutionary characters
> ```
> 
> to explain which Cyrillic characters are included

I thought about it, but in the end went with just the name of the character suite, matching the fact that Basic Greek and Polytonic Greek do not have anything there. I certainly have no objections to adding it.

Reply:
I'll plan to do that, but I'm inclined to wait until the discussion in the forums comes to some conclusion regarding the character suite itself.